src/table/components/TableHeadWrapper.jsx

```diff
@@ -45,8 +46,11 @@ function TableHeadWrapper({
   focusedCellCoord,
   setFocusedCellCoord,
   keyboard,
+  backgroundColor,
 }) {
   const headStyle = useMemo(() => getHeadStyle(layout, theme), [layout, theme.name()]);
+  // transparent is not allowed
+  headStyle.backgroundColor = backgroundColor === 'transparent' ? 'inherent' : backgroundColor;
```

Comment:
Is this only for the header? Can you have transparent on the rest of the table?

Reply:
yes, we can have transparent on the rest of the table

Reply:
why not the header?